ui/app/components/calendar-widget.js

```diff
@@ -0,0 +1,176 @@
+import Component from '@glimmer/component';
+import layout from '../templates/components/calendar-widget';
+import { setComponentTemplate } from '@ember/component';
+import { format, formatRFC3339 } from 'date-fns';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+// ARG TODO documentation takes start and end for handQuery
```

Comment:
this component will eventually have functions passed down from the parent which is why I left the TODO comment there.